README.md

```diff
@@ -108,6 +108,9 @@ are only conditionally compiled as explained below):
 
 - [libevent](https://github.com/libevent/libevent)
 - [geoip](https://github.com/maxmind/geoip-api-c)
+- [c-ares](https://github.com/c-ares/c-ares)
+- [openssl](https://github.com/openssl/openssl) or
+  [libressl](https://github.com/libressl-portable/portable)
```

Comment:
When stable is merged next into master, this needs to be moved in the Unix tutorial